### PR TITLE
Make automation

### PR DESCRIPTION
**Pull request by YoussB**
This PR:
- Adds some automation commands that I usually use to update resource types, deploy, build docker and publish docker.
Should come in handy 😁 when creating the pipeline

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+DEPLOY_DATE = $(shell date +%s)
+
+update-resources:
+	git submodule update
+	  cd resource-types && \
+	  git checkout master && \
+	  git pull
+	git add resource-types && \
```
Comment (jomsie) [inline Makefile]: looks like you cd into resource-types then try to add resource-types folder without goin' up a directory? just tried locally and it had a problem with that

Comment (YoussB) [reply Makefile]: I remember it was working on my side. Probably because `git submodule update` wasn't run in your repo?
I will give it another shot.

Comment (jomsie) [reply Makefile]: i might just be crazy (i did just copy and paste the instructions)

Comment (jomsie) [reply Makefile]: yeah it worked for me :D my bad

**Pull request merged**